pages/dkp/kommander/2.1/release-notes/2.1.1/index.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: layout.pug
+navigationTitle: Kommander 2.1.1 Release Notes
+title: Kommander 2.1.1 Release Notes
+menuWeight: 20
+excerpt: View release-specific information for Kommander 2.1.1
+enterprise: false
+beta: false
+---
+**D2iQ&reg; Kommander&reg; version 2.1.0 was released on December 23, 2021.**
+
+[button color="purple" href="https://support.d2iq.com/hc/en-us/articles/4409215222932-Product-Downloads"]Download Kommander[/button]
+
+To get started with Kommander, [download](../../download/) and [install](../../install/) the latest version of Kommander.
+
+<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install Konvoy.</p>
+
+## Release summary
+
+This release provides new features and enhancements to improve the user experience, fix reported issues, integrate changes from previous releases, and maintain compatibility and support for other packages used in Kommander.
+
+## New features and capabilities
+
+With this release, we provide you the ability to deploy or delete the following Catalog Applications from Kommander:
+
+* Kafka
+* Spark
+* Zookeeper
+
```

Comment:
@cbuto @cwyl02 - please add to & update this section as necessary. Is there anything beyond these 3 applications that we need to call out here?

Reply:
I'm not sure if any other feature were added but this is what York and I worked on. Should we specify that these are operators? since the actual kafka/spark/zookeeper apps are manually deployed by users

Reply:
I think we better use Kafka Operator, Spark Operator, Zookeeper Operator here.